jib-cli/src/main/java/com/google/cloud/tools/jib/cli/cli2/Jar.java

```diff
@@ -41,13 +42,20 @@
   @SuppressWarnings("NullAway.Init") // initialized by picocli
   private Path jarFile;
 
+  private final SingleThreadedExecutor executor = new SingleThreadedExecutor();
```

Comment:
I don't think this needs to be a field. Just instantiate as a local variable in `call()`. I know you'll going to use it in a `finally` block, so you can create it before `try {`.